### PR TITLE
feat(examples): add GitHub Enterprise Server versions

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
Adds the github enterprise server directories to the package export

Resolves #ISSUE_NUMBER

----

### Before the change?

* 

### After the change?

* 

### Pull request checklist
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

### Does this introduce a breaking change?

Please see our docs on [breaking changes](https://github.com/octokit/.github/blob/master/community/breaking_changes.md) to help!

- [ ] Yes
- [x] No

----

### CODE REVIEW

**Comment by kfcampbell** [comment]
@wolfy1339 or @gr2m I'm good with this change; do you see a reason not to ship it?

**Comment by wolfy1339** [comment]
The GHES examples aren't kept up-to-date
The GHES versions there are outdated

File: payload-examples/package.json [hunk 1]
```diff
@@ -9,7 +9,12 @@
   "main": "api.github.com/index.json",
   "files": [
     "index.d.ts",
-    "api.github.com/index.json"
+    "api.github.com/index.json",
+    "ghes-31/index.json",
+    "ghes-32/index.json",
+    "ghes-33/index.json",
+    "ghes-34/index.json",
+    "ghes-35/index.json"
```
Comment (wolfy1339) [inline payload-examples/package.json:17]: Let's keep this simple
```suggestion
    "api.github.com/index.json",
    "ghes-*/index.json"
```

**Comment by Ethan-Arrowood** [comment]
Would love to get these uptodate. How could I help?

**Review by gr2m — APPROVED** [review]
I don't mind

**Review by nickfloyd — APPROVED** [review]

**Pull request merged**